.gitignore

```diff
@@ -2,40 +2,133 @@
 logs
 *.log
 npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.pnpm-debug.log*
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
 # Runtime data
 pids
 *.pid
 *.seed
+*.pid.lock
 
 # Directory for instrumented libs generated by jscoverage/JSCover
 lib-cov
 
-# Coverage directory used by tools like nyc
+# Coverage directory used by tools like istanbul
```

Comment:
why change this? istanbul is deprecated? is this from a template?

Reply:
Yes it is, stolen from : https://github.com/github/gitignore/blob/main/Node.gitignore